<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpoi_window</class>
 <widget class="QMainWindow" name="simpoi_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simpoi_window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="poi1_group">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>poi (blue)</string>
    </property>
    <widget class="Line" name="poi1_vline">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>20</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="path1_angle_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path start angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="poi1_direction_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi direction</string>
     </property>
    </widget>
    <widget class="QSlider" name="path1_speed_slider">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="poi1_speed_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="poi1_cc_radio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>counter-clockwise</string>
     </property>
    </widget>
    <widget class="QLabel" name="po1_angle_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi start angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="path1_direction_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path direction</string>
     </property>
    </widget>
    <widget class="QSlider" name="poi1_angle_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="path1_speed_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path speed</string>
     </property>
    </widget>
    <widget class="QSlider" name="path1_angle_slider">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="poi1_speed_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi speed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="path1_cc_radio">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>counter-clockwise</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="poi1_clockwise_radio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>clockwise</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="path1_clockwise_radio">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>clockwise</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="path1_height_edit">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>210</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="path1_width_edit">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>240</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="preset1_combo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="poi1_reset_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>reset poi</string>
     </property>
    </widget>
    <widget class="QLabel" name="preset1_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>presets</string>
     </property>
    </widget>
    <widget class="QLabel" name="path1_height_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="path1_width_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path width:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="path1_center_y">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>270</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="path1_center_x">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="path1_center_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path center:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="poi2_group">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>poi (pink)</string>
    </property>
    <widget class="Line" name="poi1_vline_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>20</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="path2_angle_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path start angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="poi1_direction_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi direction</string>
     </property>
    </widget>
    <widget class="QSlider" name="path2_speed_slider">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="poi1_speed_slider_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="poi1_cc_radio_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>counter-clockwise</string>
     </property>
    </widget>
    <widget class="QLabel" name="po1_angle_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi start angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="path2_direction_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path direction</string>
     </property>
    </widget>
    <widget class="QSlider" name="poi1_angle_slider_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="path2_speed_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>path speed</string>
     </property>
    </widget>
    <widget class="QSlider" name="path2_angle_slider">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="poi1_speed_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>poi speed</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="path2_cc_radio">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>counter-clockwise</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="poi1_clockwise_radio_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>clockwise</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="path2_clockwise_radio">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>clockwise</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="path2_height_edit">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>210</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="path2_width_edit">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>240</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="preset1_combo_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="poi1_reset_button_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>reset poi</string>
     </property>
    </widget>
    <widget class="QLabel" name="preset1_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>presets</string>
     </property>
    </widget>
    <widget class="QLabel" name="path2_height_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="path2_width_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path width:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="path2_center_y">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>270</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="path2_center_x">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="path2_center_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>path center:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="poi_view" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>681</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimPoi">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Paths"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Pattern"/>
    <addaction name="actionSave_Pattern"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimPoi"/>
  </widget>
  <action name="actionLoad_Pattern">
   <property name="text">
    <string>Load Pattern</string>
   </property>
  </action>
  <action name="actionSave_Pattern">
   <property name="text">
    <string>Save Pattern</string>
   </property>
  </action>
  <action name="actionShow_Paths">
   <property name="text">
    <string>Show Paths</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
